ddtrace/profiling/profiler.py

```diff
@@ -131,7 +133,12 @@ class _ProfilerInstance(service.Service):
 
     _recorder = attr.ib(init=False, default=None)
     _collectors = attr.ib(init=False, default=None)
-    _scheduler = attr.ib(init=False, default=None)
+    _scheduler = attr.ib(
+        init=False, default=None, type=Union[scheduler.Scheduler, serverless_scheduler.ServerlessScheduler]
+    )
```

Comment:
```suggestion
    _scheduler = attr.ib(
        init=False, default=None, type=scheduler.Scheduler,
    )
```

The serverless scheduler is a subclass so that's ok

Reply:
I've tried this, when combined with the comment to remove the Union at line 225, mypy still fails:
```
ddtrace/profiling/profiler.py:222: error: Incompatible types in assignment (expression has type "Type[Scheduler]", variable has type "Type[ServerlessScheduler]")  [assignment]
Found 1 error in 1 file (checked 404 source files)
Test failed with exit code 1
```

Reply:
Can you show line 225 when you have this error?
It seems that you are assigning a class rather than instance on L225.